### PR TITLE
Develop/v5.1.0

### PR DESCRIPTION
**Pull request by ghkdqhrbals**

### CODE REVIEW

File: docker-compose-dev.yaml [hunk 1]
```diff
@@ -101,36 +101,57 @@ services:
...
+    ports:
+      - "8010:8010"
+    links:
+      - user-redis
+    environment:
+      SPRING_RABBITMQ_HOST: rabbitmq
+      SPRING_CLOUD_CONFIG_URI: http://configuration-server:8888
+      eureka_client_service-url_defaultZone: http://discovery-server:8761/eureka
+      spring_datasource_hikari_maximum-pool-size: 30
+      spring_datasource_url: jdbc:postgresql://user-db:5435/user
```
Comment (ghkdqhrbals2) [inline docker-compose-dev.yaml:116]: AWS-RDS -> Local DB

File: docker-compose-prod.yaml
Comment (ghkdqhrbals2) [inline docker-compose-prod.yaml:1]: Remove unused-production

File: shop-user-service/src/main/java/com/example/shopuserservice/OrderServiceApplication.java
Comment (ghkdqhrbals2) [inline shop-user-service/src/main/java/com/example/shopuserservice/OrderServiceApplication.java:1]: Include basPackage to fix #24

File: shop-user-service/src/main/resources/application.yaml [hunk 2]
```diff
@@ -1,4 +1,6 @@
 spring:
+  main:
+    web-application-type: reactive
```
Comment (ghkdqhrbals) [inline shop-user-service/src/main/resources/application.yaml:3]: Change type to reactive to fix #24

**Review by ghkdqhrbals — COMMENTED** [review]
Fix #24

**Review by ghkdqhrbals2 — APPROVED** [review]

**Pull request merged**